examples/vue/package.json

```diff
@@ -4,11 +4,11 @@
   "private": true,
   "devDependencies": {
     "react-scripts": "0.9.5",
-    "@storybook/vue": "^3.0.0",
+    "@storybook/vue": "3.0.0-alpha.0",
```

Comment:
@alexandrebodin can you explain a little more? i'm wondering if this is related to my problems on the kitchen-sink project.